documentation/release-notes/topics/version_4.md

```diff
@@ -1,5 +1,92 @@
+More information is in the [Microsoft documentation](https://support.microsoft.com/en-us/office/create-incoming-webhooks-with-workflows-for-microsoft-teams-8ae491c7-0394-4861-ba59-055e33f75498).
+
+:::{important}
+If you have setup notifications using the deprecated Webhook interface, please convert to the new method to avoid an
+interruption in notifications. Contact Hyperview support if you need any assistance or advice on the matter.
+:::
+
+### Enhanced Feature: Layout editor
+
+1. A new "Multi Select" menu has been added the layout header action item group to allow for group select of various layout components.
+   Capabilities have been added for select all, select all shapes, select all Racks, and select all assets as well as a rectangular
+   selection box that can be drawn on the layout. Once selected items can be removed or group edited depending on context.
+
+2. The view filter for rack total power has been proved to dynamically read the rack design value from the parent location and adjust
+   the color schema based on the setting.
+
+### Enhanced Feature: Rack power widget
+
+To allow for better power planning, The Rack Power Widget has been enhanced to show total template power for devices in the rack and
+planned power.
+
+The extra information will appear as constant lines in the graph and uses asset model information.
+
+### Enhanced Feature: Bulk actions
+
+The Bulk Actions -> Update Asset Property bulk action has been updated to add Template Power.
+
+### Enhanced Feature: Rack layout
+
+The rack layout has been enhanced to allow customers to place assets in the front and rear of the rack in the same rack unit.
+
+### Enhanced Feature: Asset power sensors
+
+The power analyzer will now take into consideration other available power information such as metered outlets connected to the asset
+and will calculate power metrics where applicable.
+
+:::{note}
+This will add new computed sensors to track this data.
+:::
+
+### Enhanced Feature: Discovery
+
+The discovery engine has been enhanced to support more asset variations from vendors like Vertiv/Liebert, Schneider Electric/APC and others. Customers are required to have the latest version of the Linux Data Collector to take advantage of these improvements.
+
+### Other notable changes and improvements
+
+- The Camera asset type can now be added manually. Prior to this, the Camera asset type could only be discovered.
+- The Circuit Name property with-in connectivity has been renamed to Circuit ID to be more inline with industry conventions.
+- The Template Power asset property can now be edited by users regardless of the source.
+- The notification event template has been updated to provide more information.
+- Bulk import validation has been improved with more extensive data validation, and more clear error messages.
+- Improved access policy checks when using layout editor.
+- Improved access policy checks when using the Carbon footprint report.
```

Comment:
"Carbon footprint" should be "Carbon Footprint".

Reply:
Fixed, thanks!